cinema/tests/test_movie_api.py

```diff
@@ -1,16 +1,23 @@
 import tempfile
+
 import os
 
 from PIL import Image
+
 from django.contrib.auth import get_user_model
+
 from django.test import TestCase
+
 from django.urls import reverse
 
 from rest_framework.test import APIClient
+
 from rest_framework import status
 
 from cinema.models import Movie, MovieSession, CinemaHall, Genre, Actor
 
+from cinema.serializers import MovieListSerializer, MovieDetailSerializer
+
```

Comment:
why do you need to separate each import with empty line?

Reply:
In my opinion, it looks better this way.